### PR TITLE
Refactored review pages

### PR DESCRIPTION
**Pull request by Steve-Leke**
We changed all the paths within the reviewPage.html file.

Fixed #23

### CODE REVIEW

File: FirebaseRTC/public/reviewPage.html [hunk 1]
```diff
@@ -23,10 +23,10 @@
 
 
     
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" type="text/css" href="../../assets/css/bootstrap.min.css">
```
Comment (gideoncodenext) [inline FirebaseRTC/public/reviewPage.html]: In the future, make a habit to use direct terms/names of directories and sub directories. An example of specific file pathway directions can be found here: https://github.com/mellychat/webapp/blob/6ee49c0ae171062c6f67f7218d7094988652a327/public/templates/patientSignUp.html#L18
Make similar file pathway specifications for the other hrefs in the file.

Coach joe says do the thing...with the thing

File: FirebaseRTC/public/patWaitingRoom.html [hunk 2]
```diff
@@ -23,10 +23,10 @@
 
     <link rel="stylesheet" type="text/css" href="prWaitingRoom.css">
   <link rel="stylesheet" type="text/css" href="patWaitingRoom.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" type="text/css" href="/assets/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="/assets/css/font-awesome.css">
+    <link rel="stylesheet" type="text/css" href="/assets/csslatemo-art-factory.css">
```
Comment (gideoncodenext) [inline FirebaseRTC/public/patWaitingRoom.html]: "/temp" was lost from "css/templatemo-art-factory.css" Be careful with find and replace function.

File: FirebaseRTC/public/prWaitingRoom.html [hunk 3]
```diff
@@ -22,10 +22,10 @@
 
      <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="prWaitingRoom.css">
-     <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+     <link rel="stylesheet" type="text/css" href="/assets/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="/assets/css/font-awesome.css">
+    <link rel="stylesheet" type="text/css" href="/assets/csslatemo-art-factory.css">
```
Comment (gideoncodenext) [inline FirebaseRTC/public/prWaitingRoom.html]: Same issue with "/temp" being lost in the file pathway change. See comment in patWaitingRoom.html file

File: FirebaseRTC/public/reviewPage.html [hunk 4]
```diff
@@ -47,14 +47,14 @@
                 <div class="col-12">
                     <nav class="main-nav">
                         <!-- ***** Logo Start ***** -->
-                        <a href="/index.html" class="logo">Melly</a>
```
Comment (gideoncodenext) [inline FirebaseRTC/public/reviewPage.html]: Deleted "/" looks sus... maybe check this thing

Comment (gideoncodenext) [reply FirebaseRTC/public/reviewPage.html]: I noticed that this deleted "/" looks inconsistent with the other changes made so far.

**Review by gideoncodenext — DISMISSED** [review]
All of the refactoring looks good, however we need to make sure that the files all end up under the public directory and we no longer reference file pathways with ".." or "../.." etc. Check out this link: https://github.com/mellychat/webapp/projects 

For the ideal file setup and structure. Thanks!

File: FirebaseRTC/public/reviewPage.html (hunk 1 above)
Comment (Steve-Leke) [reply FirebaseRTC/public/reviewPage.html]: The issue is fixed in the new commit.

File: FirebaseRTC/public/patWaitingRoom.html (hunk 2 above)
Comment (Steve-Leke) [reply FirebaseRTC/public/patWaitingRoom.html]: The issue is fixed in the new commit.

File: FirebaseRTC/public/prWaitingRoom.html (hunk 3 above)
Comment (Steve-Leke) [reply FirebaseRTC/public/prWaitingRoom.html]: The issue is fixed in the new commit.

File: FirebaseRTC/public/reviewPage.html (hunk 4 above)
Comment (Steve-Leke) [reply FirebaseRTC/public/reviewPage.html]: This issue is fixed in the new commit.

File: FirebaseRTC/public/patWaitingRoom.html (hunk 2 above)
Comment (lillianchou94) [reply FirebaseRTC/public/patWaitingRoom.html]: Line 28 is still incorrect. 
Should be:
`<link rel="stylesheet" type="text/css" href="/assets/css/templatemo-art-factory.css">`

Comment (lillianchou94) [reply FirebaseRTC/public/patWaitingRoom.html:28]: <link rel="stylesheet" type="text/css" href="/assets/css/templatemo-art-factory.css">

**Review by lillianchou94 — DISMISSED** [review]

**Review by lillianchou94 — APPROVED** [review]

**Pull request merged**